spec/search_paths_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'spec_helper'
+require 'ronin/wordlists/search_paths'
+require 'ronin/wordlists/exceptions'
+
+describe Ronin::Wordlists::SearchPaths do
+  subject { described_class.new([path]) }
+
+  let(:path)         { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures', 'wordlists')) }
+  let(:wordlist_dir) { Ronin::Wordlists::WordlistDir.new(path) }
+
+  describe "#initialize" do
+    it "must initialize #paths and call #<< for each one" do
+      expect(subject.paths[0]).to be_kind_of(Ronin::Wordlists::WordlistDir)
+      expect(subject.paths[0].path).to eq(path)
+    end
+  end
+
+  describe ".[]" do
+    it "must initialize SearchPaths instance" do
+      expect(described_class[path]).to be_kind_of(Ronin::Wordlists::SearchPaths)
+      expect(described_class[path].paths[0].path).to eq(path)
```

Comment:
Probably should extract the repeated `described_class[path]` into `subject { described_class[path] }`.